packages/blade/src/components/ThemeProvider/useTheme.ts

```diff
@@ -0,0 +1,23 @@
+import { useContext, createContext } from 'react';
+import { lightTheme, Theme } from '../../tokens/theme';
+import { UseColorScheme } from '../../utils/useColorScheme';
+
+export type ThemeContext = {
+  theme: Theme;
+} & UseColorScheme;
+
+export const ThemeContext = createContext<ThemeContext>({
+  theme: lightTheme,
+  colorScheme: 'light',
+  setColorScheme: () => '',
```

Comment:
Any reason to return string?

```suggestion
  setColorScheme: () => null,
```

Reply:
good catch.